<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_tableeditor</class>
 <widget class="QDialog" name="Dialog_tableeditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Werte anpassen</string>
  </property>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>390</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="listWidget_zeilen"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_new">
      <property name="text">
       <string>neu</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_del">
      <property name="text">
       <string>entf</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>571</width>
     <height>347</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="2,3">
    <item row="1" column="0">
     <widget class="QComboBox" name="comboBox_param2"/>
    </item>
    <item row="0" column="0">
     <widget class="QComboBox" name="comboBox_param"/>
    </item>
    <item row="2" column="0">
     <widget class="QComboBox" name="comboBox_param3"/>
    </item>
    <item row="6" column="0">
     <widget class="QComboBox" name="comboBox_param7"/>
    </item>
    <item row="4" column="0">
     <widget class="QComboBox" name="comboBox_param5"/>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="comboBox_param4"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_value"/>
    </item>
    <item row="5" column="0">
     <widget class="QComboBox" name="comboBox_param6"/>
    </item>
    <item row="7" column="0">
     <widget class="QComboBox" name="comboBox_param8"/>
    </item>
    <item row="8" column="0">
     <widget class="QComboBox" name="comboBox_param9"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_value2"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_value3"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_value4"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_value5"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_value6"/>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEdit_value7"/>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEdit_value8"/>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="lineEdit_value9"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>listWidget_zeilen</tabstop>
  <tabstop>lineEdit_value</tabstop>
  <tabstop>lineEdit_value2</tabstop>
  <tabstop>lineEdit_value3</tabstop>
  <tabstop>lineEdit_value4</tabstop>
  <tabstop>lineEdit_value5</tabstop>
  <tabstop>lineEdit_value6</tabstop>
  <tabstop>lineEdit_value7</tabstop>
  <tabstop>lineEdit_value8</tabstop>
  <tabstop>lineEdit_value9</tabstop>
  <tabstop>comboBox_param</tabstop>
  <tabstop>comboBox_param2</tabstop>
  <tabstop>comboBox_param3</tabstop>
  <tabstop>comboBox_param4</tabstop>
  <tabstop>comboBox_param5</tabstop>
  <tabstop>comboBox_param6</tabstop>
  <tabstop>comboBox_param7</tabstop>
  <tabstop>comboBox_param8</tabstop>
  <tabstop>comboBox_param9</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>pushButton_new</tabstop>
  <tabstop>pushButton_del</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
